src/main.rs

```diff
@@ -81,202 +50,8 @@ https://bitbucket.org/user/repo
 
     let src = matches.value_of("src").unwrap();
 
-    let repo = parse(src);
-    let repo = repo.unwrap();
-
-    let dest = matches
-        .value_of("dest")
-        .map_or(std::env::current_dir().unwrap().join(&repo.project), |x| {
```

Comment:
This should probably be kept to have a fallback when no destination is specified

Reply:
so I tried reaching that code-branch, but due to the validator `validator(degit::validate_dest)` there is no way that the destination cannot be specified.. so I removed it because it also would've needed the repo there 😄